saleor/core/utils/__init__.py

```diff
@@ -153,3 +153,18 @@ def prepare_unique_attribute_value_slug(attribute: "Attribute", slug: str):
         "slug", flat=True
     )
     return prepare_unique_slug(slug, value_slugs)
+
+
+def queryset_in_batches(queryset, batch_size):
+    """Slice a queryset into batches.
```

Comment:
Can you add here a note that this should be used only with the objects that have `int` as a `pk`?

Reply:
Why only integers? You can compare strings, dates as well. ie. `Product.objects.filter(name__gt="abc")` is a valid query, right?

Reply:
Nevermind. I messed up this with something different 😅